### PR TITLE
Build .rpm and .deb packages

### LINKED ISSUE #734: Build .rpm and .deb packages
We should provide `.rpm` and `.deb` packages. Let's build them with https://nfpm.goreleaser.com.

We should make a `task` target `release-build-packages` that would build packages into `build/rpm` and `build/deb` directories (that should be in `.gitignore`). That target should be stable, so the external package maintainers can rely on it. That target should be mentioned in the documentation.

We should also update our release checklist to include a step where we would upload those packages to release.

Automatic upload and/or releasing is not in scope.

### PR DESCRIPTION
**Pull request by fenogentov**
Closes #734.

### CODE REVIEW

File: Taskfile.yml [hunk 1]
```diff
@@ -276,3 +276,17 @@ tasks:
         sh: cat internal/util/version/commit.txt
       VERSION:
         sh: cat internal/util/version/version.txt
+
+  nfpm:
+    cmds:
+      - mkdir -p ./build/deb
+      - mkdir -p ./build/rpm
```
Comment (AlekSi) [inline Taskfile.yml:283]: That could be done using `dir:`: https://taskfile.dev/usage/#task-directory

File: Taskfile.yml [hunk 2]
```diff
@@ -276,3 +276,17 @@ tasks:
...
+      - nfpm pkg -f .nfpm.yaml -t ./build/deb/ferretdb_{{.VERSION}}.deb
```
Comment (AlekSi) [inline Taskfile.yml:284]: Let's install `nfpm` in tools.go

**Review by AlekSi — CHANGES_REQUESTED** [review]

File: Taskfile.yml [hunk 3]
```diff
@@ -283,3 +283,38 @@ tasks:
...
+    deps: [build]
+    vars:
+      VERSION:
+        sh: cat ../../internal/util/version/version.txt
+    dir: build/rpm
+    cmds:
+      - ../../bin/nfpm pkg --config ../../.nfpm.yml --packager rpm --target ferretdb_${VERSION}.rpm
+    env:
+      VERSION:
+        sh: cat ../../internal/util/version/version.txt | grep -oP '^(.+?)(?=-)'
```
Comment (AlekSi) [inline Taskfile.yml:315]:
```
task: Failed to run task "packager": task: Command "cat ../../internal/util/version/version.txt | grep -oP '^(.+?)(?=-)'" failed: exit status 2
```

Let's wait until #798 is merged, then change generate.go to generate another file with a clean version

**Review by AlekSi — COMMENTED** [review]
https://github.com/FerretDB/FerretDB/pull/739#discussion_r914846510

**Comment by fenogentov** [comment]
> [#739 (comment)](https://github.com/FerretDB/FerretDB/pull/739#discussion_r914846510)

#739 is `Merged`
it got me confused

**Comment by AlekSi** [comment]
Actually, why we need `grep` at all? Version generated by `git describe --tags --dirty` should be good enough

**Comment by fenogentov** [comment]
> Actually, why we need `grep` at all? Version generated by `git describe --tags --dirty` should be good enough

We are satisfied with the file "ferretdb_v0.4.0-56-g15af921-dirty.deb"?
Or is it "ferretdb_v0.4.0-56-g15af921.deb"?

File: Taskfile.yml [hunk 4]
```diff
@@ -287,3 +287,33 @@ tasks:
...
+
+
+  nfpm_deb:
+    deps: [build]
+    dir: build/deb
+    cmds:
+      - ../../bin/nfpm pkg --config ../../.nfpm.yml --packager deb --target ferretdb_${FERRETDB_VERSION}.deb
+    env:
+      FERRETDB_VERSION:
+        sh: git describe --tags --dirty
```
Comment (AlekSi) [inline Taskfile.yml:304]:
```suggestion
      VERSION:
        sh: cat internal/util/version/gen/version.txt
```

Comment (AlekSi) [reply Taskfile.yml:304]: The same what we use above

File: Taskfile.yml [hunk 5]
```diff
@@ -287,3 +287,33 @@ tasks:
         sh: cat internal/util/version/gen/version.txt
       COMMIT:
         sh: cat internal/util/version/gen/commit.txt
+        
+  build:
+    deps: [gen-version]
+    cmds:
+      - go build -v -o ./bin/ ./cmd/ferretdb
```
Comment (AlekSi) [inline Taskfile.yml:294]: We should use existing `build-testcover`

**Review by AlekSi — COMMENTED** [review]
Let's also tweak CI config to build package there, just for testing for now, without actually uploading them anywhere

**Review by AlekSi — APPROVED** [review]

**Comment by AlekSi** [comment]
Thank you!

**Pull request merged**